sql/driver/wire.proto

```diff
@@ -71,6 +71,11 @@ message Response {
     // presence indicates that the operation was a DDL statement.
     message DDL {}
 
+    // Ack is a message containing the tag of a result without a meaningful return.
+    message Ack {
```

Comment:
It feels wrong to be adding stuff to the protobuf-protocol in order to support features in the pgwire protocol. I think it would be worth the effort to remove the use of these protos from the pgwire implementation.